src/Integration.UnitTests/Connection/ConnectionWorkflowTests.cs

```diff
@@ -33,6 +33,7 @@
 using SonarLint.VisualStudio.Core.CFamily;
 using SonarLint.VisualStudio.Core.JsTs;
 using SonarLint.VisualStudio.Infrastructure.VS;
+using SonarLint.VisualStudio.Integration.Binding;
```

Comment:
Why is this file changed?

Reply:
this is also a merge mistake

Reply:
it has a reference to IJsTsProjectTypeIndicator. Because we have moved it, its interface has changed.